snmp/datadog_checks/snmp/data/default_profiles/ruckus-unleashed.yaml

```
@@ -0,0 +1,78 @@
+extends:
+  - _base.yaml
+  - _generic-if.yaml
+  - ruckus.yaml
+
+sysobjectid:
+  - 1.3.6.1.4.1.25053.3.1.5.*  # Ruckus ZoneDirector
+  - 1.3.6.1.4.1.25053.3.1.5.1  # Ruckus ZD1000
+  - 1.3.6.1.4.1.25053.3.1.5.2  # Ruckus ZD1100
+  - 1.3.6.1.4.1.25053.3.1.5.3  # Ruckus ZD3000
+  - 1.3.6.1.4.1.25053.3.1.5.8  # Ruckus ZD5000
+  - 1.3.6.1.4.1.25053.3.1.5.15  # Ruckus ZD1200
+metrics:
+  - MIB: RUCKUS-UNLEASHED-SYSTEM-MIB
+    symbol:
+      name: cpu.usage
+      OID: 1.3.6.1.4.1.25053.1.15.1.1.1.15.13.0
+  - MIB: RUCKUS-UNLEASHED-SYSTEM-MIB
+    symbol:
+      name: memory.usage
+      OID: 1.3.6.1.4.1.25053.1.15.1.1.1.15.14.0
+  - MIB: RUCKUS-UNLEASHED-SYSTEM-MIB
+    symbol:
+      name: ruckusUnleashedSystemStatsNumRegisteredAP
+      OID: 1.3.6.1.4.1.25053.1.15.1.1.1.15.15.0
+  - MIB: RUCKUS-UNLEASHED-SYSTEM-MIB
+    symbol:
+      name: ruckusUnleashedSystemStatsNumAP
+      OID: 1.3.6.1.4.1.25053.1.15.1.1.1.15.1.0
+  - MIB: RUCKUS-UNLEASHED-SYSTEM-MIB
+    symbol:
+      name: ruckusUnleashedSystemStatsWLANTotalRxPkts
+      OID: 1.3.6.1.4.1.25053.1.15.1.1.1.15.5.0
+    metric_type: monotonic_count
+  - MIB: RUCKUS-UNLEASHED-SYSTEM-MIB
+    symbol:
+      name: ruckusUnleashedSystemStatsWLANTotalRxBytes
+      OID: 1.3.6.1.4.1.25053.1.15.1.1.1.15.6.0
+    metric_type: monotonic_count
+  - MIB: RUCKUS-UNLEASHED-SYSTEM-MIB
+    symbol:
+      name: ruckusUnleashedSystemStatsWLANTotalTxPkts
+      OID: 1.3.6.1.4.1.25053.1.15.1.1.1.15.8.0
+    metric_type: monotonic_count
+  - MIB: RUCKUS-UNLEASHED-SYSTEM-MIB
+    symbol:
+      name: ruckusUnleashedSystemStatsWLANTotalTxBytes
+      OID: 1.3.6.1.4.1.25053.1.15.1.1.1.15.9.0
+    metric_type: monotonic_count
+  - MIB: RUCKUS-UNLEASHED-SYSTEM-MIB
+    symbol:
+      name: ruckusUnleashedSystemStatsWLANTotalTxFail
+      OID: 1.3.6.1.4.1.25053.1.15.1.1.1.15.11.0
+    metric_type: monotonic_count
+  - MIB: RUCKUS-UNLEASHED-SYSTEM-MIB
+    symbol:
+      name: ruckusUnleashedSystemStatsWLANTotalTxRetry
+      OID: 1.3.6.1.4.1.25053.1.15.1.1.1.15.12.0
+    metric_type: monotonic_count
+  - MIB: RUCKUS-UNLEASHED-SYSTEM-MIB
+    symbol:
+      name: ruckusUnleashedSystemStatsWLANTotalAssocFail
+      OID: 1.3.6.1.4.1.25053.1.15.1.1.1.15.16.0
+    metric_type: monotonic_count
+  - MIB: RUCKUS-UNLEASHED-SYSTEM-MIB
+    symbol:
+      name: ruckusUnleashedSystemStatsAllNumSta
+      OID: 1.3.6.1.4.1.25053.1.15.1.1.1.15.30.0
+metric_tags:
+  - OID: 1.3.6.1.4.1.25053.1.15.1.1.1.1.9.0
+    symbol: ruckusUnleashedSystemModel
+    tag: ruckus_unleashed_system_model
+  - OID: 1.3.6.1.4.1.25053.1.15.1.1.1.1.12.0
+    symbol: ruckusUnleashedSystemLicensedAPs
+    tag: ruckus_unleashed_system_licensed_a_ps
```

```suggestion
    tag: ruckus_unleashed_system_licensed_aps
```

Since it's a bit weird to split after `APs` into two parts like `a_ps`. APs, means Access Point (s)